src/routes/RemediationDetails.js

```diff
@@ -201,19 +204,17 @@ const RemediationDetails = ({
             </LevelItem>
             <LevelItem>
               <Split hasGutter>
-                {context.hasSmartManagement && (
-                  <SplitItem>
-                    <ExecutePlaybookButton
-                      isDisabled={
-                        !context.isReceptorConfigured ||
-                        !context.permissions.execute ||
-                        !context.hasSmartManagement
-                      }
-                      disabledStateText={getDisabledStateText()}
-                      remediationId={remediation.id}
-                    ></ExecutePlaybookButton>
-                  </SplitItem>
-                )}
+                <SplitItem>
+                  <ExecutePlaybookButton
+                    isDisabled={
+                      !context.isReceptorConfigured ||
```

Comment:
In the case of RHC we don't actually need receptor to be configured anymore in order to run Playbooks.  @PanSpagetka can we remove the `isReceptorConfigured` check from this button?